src/main/java/org/distributed/systems/ChordStart.java

```diff
@@ -4,17 +4,53 @@
 import akka.actor.ActorSystem;
 import akka.actor.Props;
 import org.distributed.systems.chord.actors.Node;
+import org.distributed.systems.chord.messaging.*;
+import org.distributed.systems.chord.model.ChordNode;
+import org.distributed.systems.chord.util.IHashUtil;
+import org.distributed.systems.chord.util.impl.HashUtil;
 
 public class ChordStart {
 
     public static void main(String[] args) {
+        IHashUtil hashUtil = new HashUtil();
+
+        // Create actor system
         ActorSystem system = ActorSystem.create("ChordNetwork"); // Setup actor system
-        ActorRef node = system.actorOf(Props.create(Node.class)); // Create new actor: node
 
-        node.tell("JOIN", ActorRef.noSender()); // Send message
+        // Create start node
+        ChordNode startNode = new ChordNode(0L);
+        final ActorRef node = system.actorOf(Props.create(Node.class), "ChordActor0");
+
+        // Create messages
+        NodeJoinMessage joinMessage = new NodeJoinMessage(startNode);
+        NodeLeaveMessage leaveMessage = new NodeLeaveMessage(startNode);
+        GetFingerTableMessage getFingerTableMessage = new GetFingerTableMessage();
+
+        String hashKey = hashUtil.hash(String.valueOf(startNode.getId()));
+        PutValueMessage putValueMessage = new PutValueMessage(hashKey, "TEST VALUE");
+        GetValueMessage getValueMessage = new GetValueMessage(hashKey);
+
+        // Send messages to the node
+        node.tell(getFingerTableMessage, ActorRef.noSender());
```

Comment:
this will have to be an ask instead of tell, i think? https://doc.akka.io/docs/akka/current/typed/interaction-patterns.html#request-response-with-ask-between-two-actors

Reply:
Also can we add a value this message? Then we should provide the generated hash (key of this node)

Reply:
Indeed it should be an ask. But this main class isn't an actor so I can't implemented here. And yes we can add whatever value we want, we define the messages our self. I will add some field to it.